README.md

```diff
@@ -1,56 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Conference Page
 
+> Book Records app
```

Comment:
- [ ] Kindly add a description of the project for a clear understanding of what the project is about.